lib/terraorg/version.rb

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 
 module Terraorg
-  VERSION = '0.4.0'
+  VERSION = '0.5.0'
```

Comment:
not sure where your comment went - but if you leave it like this it'll def work, thanks